### PR TITLE
Add Documentation for new Model Fitting and Corrections tabs

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR adds documentation pages for the new Model fitting tab, and Corrections tab in the Muon interfaces. It also updates the Fitting and Sequential Fitting documentation pages, and adds better release notes for these new features.

**To test:**
Spell and grammar check
Build docs-html
Make sure the new links work
Make sure the images are displayed as expected 

*There is no associated issue.*

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/muon/muon_corrections_tab.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+**Dead Time stats** Displays the minimum, maximum and average deadtime correction for the currently selected runs.
+
+Background Corrections
+^^^^^^^^^^^^^^^^^^^^^^
+
+**Background** There are three background correction options:
+
+* ``None``: No background correction should be applied.
+* ``Auto``: The background is automatically calculated by performing a fit on the loaded data before being subtracted from the data. The diagram below describes this process.
```
Comment (sf1919) [inline docs/source/interfaces/muon/muon_corrections_tab.rst:33]: The placement of the diagram is more to the side than below. Perhaps this should be reworded?

File: docs/source/interfaces/muon/muon_model_fitting_tab.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+.. image::  ../../images/muon_interface_tab_fitting_browser.png
+   :height: 125px
+
+Right clicking in the central box will bring up a context menu for adding or removing functions.
+
+**Property** This is a list of property names, individual functions can be collapsed down for greater legibility.
+
+**Value** This lists the initial guesses for variables within fitting functions.Variables can be fixed for a given fit
+from the edit paramiter values dialog box (the ellipsis to the right of the value box).
```
Comment (sf1919) [inline docs/source/interfaces/muon/muon_model_fitting_tab.rst:39]: The image for the Fit function browser does not show any ellipsis to the right of the value box. Is it possible to have an image that includes these as they are specifically referred to?

**Review by sf1919 — CHANGES_REQUESTED** [review]
The changes to the documentation are well written and very clear. They are in keeping with the style that has been developed for these.

I have a couple of queries about wording and images but these are relatively minor. See specific comments for more details.

**Comment by robertapplin** [comment]
Thanks for the review, I have made the suggested changes

**Review by sf1919 — APPROVED** [review]
I am very happy with the changes that have been made. As stated previously the documentation is clear and easy to understand. I approve this PR.

**Pull request merged**